packages/client/src/rtc/Publisher.ts

```diff
@@ -384,9 +384,10 @@ export class Publisher extends BasePeerConnection {
         const { sdp: baseSdp = '' } = offer;
         const { dangerouslyForceCodec, fmtpLine } =
           this.clientPublishOptions || {};
-        const sdp = dangerouslyForceCodec
+        let sdp = dangerouslyForceCodec
           ? removeCodecsExcept(baseSdp, dangerouslyForceCodec, fmtpLine)
           : baseSdp;
+        sdp = setStartBitrate(sdp, 1000);
```

Comment:
Ideally, the `startBitrate` parameter should be a fraction of the `target_bitrate` defined in the `publishOption`. This will prevent us from running into situations where:
- target_bitrate: 500kb, start bitrate: 1000kb
- target_bitrate: 5000kb, start bitrate: 1000kb;

Maybe start with 60/70% of the target bitrate?